git.go

```diff
@@ -121,12 +122,32 @@ func (r *BumperRepo) getModifiedFiles(last bool) ([]string, error) {
 		}
 
 		if last {
-			break
+			return errLastCommit
+		}
+		if strings.TrimSpace(commit.Author.Name) == r.name {
+			return errBumpFound
```

Comment:
instead of creating custom error, you could write log info and return `storer.ErrStop`
and later do clean
`
	if err != nil {
		return nil, err
	}
`

Reply:
done, tnx!